packages/libs/eda/src/lib/map/analysis/mapTypes/types.ts

```diff
@@ -0,0 +1,69 @@
+import { ComponentType } from 'react';
+import {
+  AnalysisState,
+  Filter,
+  PromiseHookState,
+  StudyEntity,
+} from '../../../core';
+import { GeoConfig } from '../../../core/types/geoConfig';
+import { ComputationAppOverview } from '../../../core/types/visualization';
+import { AppState } from '../appState';
+import { EntityCounts } from '../../../core/hooks/entityCounts';
+
+export interface MapTypeConfigPanelProps {
+  apps: ComputationAppOverview[];
+  analysisState: AnalysisState;
+  appState: AppState;
+  studyId: string;
+  filters: Filter[] | undefined;
+  studyEntities: StudyEntity[];
+  geoConfigs: GeoConfig[];
+  configuration: unknown;
+  updateConfiguration: (configuration: unknown) => void;
+  hideVizInputsAndControls: boolean;
```

Comment:
Are these two (*hideVizInputsAndControls) needed in the config panel?

Reply:
I'm not sure, but they were a part of the interface from a merge from `main`. I considered removing it, but didn't want to make an unnecessary interface change during a merge conflict resolution. I'm inclined to leave it for now, with the assumption that we will tweak some of the details of this feature.